### DISCUSSION TITLE
Keeping Custom CSS when Migrating from Vuetify 1.0 to the latest stable version 2.6

### DISCUSSION THREAD

**Discussion by TeraFumba** [vuetifyjs/vuetify#15124] [General]
Hi all,

I have been trying to migrate from vuetify 1.0 to 2.6 
issues: 
I couldn't keep the original look of the application vuetify over powers Custom css , I have tried the work arounds i find online no luck. can anyone point me to the right direction.

Im using webpack 4.6.0
`    rules: [
        {
          test: /\.(sass|less|css)$/,
          use: [
            'vue-style-loader',
            {
              loader: 'css-loader',
              // Requires sass-loader@^7.0.0
              options: {
                implementation: require('sass'),
                indentedSyntax: true // optional
              },
            },
          ],
        },
    ]`

On my App.vue file i am using the `v-app ` tags

**Comment by KaelWD** [comment]
You will have to rewrite your custom styles, there were lots of changes to component structure between versions.

**Reply by TeraFumba** [reply]
@KaelWD Thanks for your quick response, any docs/direction as to how i can over come this
